backend/services/container_discovery_service.py

```
@@ -0,0 +1,186 @@
+import logging
+from typing import Dict, List, Optional
+from ..models import (
+    ContainerDiscoveryResult, ContainerAnalysis, ContainerSummary, 
+    NetworkSummary, IdentifierType, VolumeMount
+)
+from ..docker_ops import DockerOperations, ContainerInfo
```

Ruff (0.11.9)

`..docker_ops.ContainerInfo` imported but unused

Remove unused import: `..docker_ops.ContainerInfo`

(F401)